server/src/error.rs

```diff
@@ -132,12 +134,16 @@ impl Display for EdgeError {
                     f,
                     "Could not fetch client features because api key was not allowed"
                 ),
+                FeatureError::NotFound => write!(
+                    f,
+                    "Could not fetch features because upstream url was not found"
```

Comment:
Could this one be a bit misleading between "No upstream URL was set" vs "We got a 404 back from the the upstream URL that you set"?

Reply:
No upstream URL is not possible. Clap handles the fact that you have to set an upstream URL if you're running in Edge mode

Reply:
Yep, I suspected that would be the case 👍